main_keysync.go

```diff
@@ -140,9 +142,11 @@ func main() {
 	logrus.Printf("Private key files will be persisted with %v permissions",
 		ksc.KeyFilePermissions)
 
-	logrus.Printf("Private key files will be persisted with %d:%d ownership (this might not work unless running as root or having CAP_CHOWN)",
-		ksc.KeyFileOwnerUID,
-		ksc.KeyFileOwnerGID)
+	if ksc.KeyFileOwnerUID != nil || ksc.KeyFileOwnerGID != nil {
```

Comment:
i would prefer && here since we are using both variables within the loop. EIther case, doesn't really matter since it is set together anyway

Reply:
changed